packages/client/modules/demo/ClientGraphQLServer.ts

```diff
@@ -243,12 +247,50 @@ class ClientGraphQLServer extends (EventEmitter as GQLDemoEmitter) {
         }
       }
     },
-    TaskFooterIntegrateMenuRootQuery: () => {
+    TaskFooterIntegrateMenuRootQuery: (_$teamId, $userId) => {
```

Comment:
-1 Please change to `(_teamId: unknown, userId: string)`. We don't use $ in parameters/variables and want to have arguments typed moving forward. Since `_teamId` is unused, it's best to type it as `unknown`.